template/gitignore

```diff
@@ -0,0 +1,7 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
Can you remove the leading `/`, it looks like you're not putting it for npm-debug.log